local-volume/provisioner/pkg/common/common.go

```diff
@@ -169,26 +176,58 @@ func GetDefaultVolumeConfig() map[string]MountConfig {
 // VolumeConfigToConfigMapData converts volume config to configmap data.
 func VolumeConfigToConfigMapData(config map[string]MountConfig) (map[string]string, error) {
 	configMapData := make(map[string]string)
-	for class, data := range config {
-		var val []byte
-		var err error
-		if val, err = json.Marshal(data); err != nil {
-			return nil, fmt.Errorf("unable to unmarshal config for class %v: %v", class, err)
-		}
-		configMapData[class] = string(val)
+	val, err := yaml.Marshal(config)
+	if err != nil {
+		return nil, fmt.Errorf("unable to Marshal volume config: %v", err)
 	}
+	configMapData[ProvisonerStorageClassConfig] = string(val)
 	return configMapData, nil
 }
 
 // ConfigMapDataToVolumeConfig converts configmap data to volume config
 func ConfigMapDataToVolumeConfig(data map[string]string) (map[string]MountConfig, error) {
```

Comment:
Is this function still needed?  Does `LoadProvisionerConfigs` accomplish this?

Reply:
no need will remove it